www/503.html

```diff
@@ -42,8 +42,14 @@
     </header>
     <div class="govuk-width-container">
       <main class="govuk-main-wrapper" id="main-content">
-        <h1 class="govuk-heading-xl">Page Unavailable</h1>
-        <p class="govuk-body">We're experiencing technical difficulties. Please try again later.</p>
+        <h1 class="govuk-heading-xl">Planned Maintenance</h1>
+        <p class="govuk-body">
+          This page is currently unavailable.<br><br>
+
+          GOV.UK is currently undergoing scheduled maintenance.<br><br>
+
+          Normal service will resume at 6:00am.
```

Comment:
_Edit:_ 6:00pm is in correct format according to https://www.gov.uk/guidance/style-guide/a-to-z#times

```suggestion
          Normal service will resume at 6:00pm.
```